docs/core/tools/global-json.md

```diff
@@ -1,41 +1,64 @@
 ---
 title: global.json reference
-description: See the schema for the global.json file, which permits setting the .NET Core tools version.
-keywords: .NET, .NET Core
-author: blackdwarf
+description: See the schema for the global.json file, which permits setting the .NET Core SDK version.
+author: mairaw
 ms.author: mairaw
-ms.date: 04/05/2017
+ms.date: 03/19/2018
 ms.topic: article
 ms.prod: .net-core
 ms.technology: dotnet-cli
-ms.devlang: dotnet
-ms.assetid: 96102f96-d403-4385-8ef6-5d80e406eb0c
 ms.workload: 
   - dotnetcore
 ---
-
 # global.json reference
 
-The *global.json* file allows selection of the .NET Core tools version being used through the `sdk` property.
+[!INCLUDE [topic-appliesto-net-core-all](../../../includes/topic-appliesto-net-core-all.md)]
+
+The *global.json* file allows selection of the .NET Core SDK version being used through the `sdk` property.
 
-.NET Core CLI tools look for this file in the current working directory (which isn't necessarily the same as the project directory) or one of its parent directories.
+.NET Core SDK looks for this file in the current working directory (which isn't necessarily the same as the project directory) or one of its parent directories.
 
 ## sdk
+
 Type: Object
 
-Specifies information about the SDK.
+Specifies information about the .NET Core SDK.
 
 ### version
+
 Type: String
 
-The version of the SDK to use.
+The version of the .NET Core SDK to use.
 
-For example:
+Note that this field:
+
+- Doesn't have globbing support, that is, the full version number has to be specified.
+- Doesn't support version ranges.
+
+The following example shows the contents of a *global.json* file:
 
 ```json
 {
   "sdk": {
-    "version": "1.0.0-preview2-003121"
+    "version": "2.1.300-preview1-008174"
   }
 }
 ```
+
+You can easily create this file by executing the [dotnet new](dotnet-new) command as follows:
+
+```console
+dotnet new globaljson --sdk-version 2.1.300-preview1-008174
+```
+
+## Matching rules
+
+With .NET Core SDK 1.x, if you specified a version and no exact match was found, the latest installed SDK version was used. Latest SDK version can be either release or pre-release - the highest version number wins.
+
+Starting with .NET Core SDK 2.0, the following rules apply when determining which version of the SDK to use:
+
+- If *global.json* doesn't contain a version or a *global.json* file cannot be found, the latest installed SDK version is used (release or pre-release).
+- If *global.json* does contain a version:
+  - If the specified version is found on the machine, that exact version is used.
+  - If the specified version cannot be found on the machine, the latest **patch** of the specified version is used (roll-forward along patch versions). For more information about versioning, see [Semantic versioning](../versions/index.md/#semantic-versioning).
```

Comment:
We aren't really using semantic versioning for the SDK starting in 2.1.300. We follow that intention with 2.1.xyy

@livarcocc or @dsplaisted should confirm but the intention is that we will roll forward in the last two digits, but the user might think the whole thing (300) is the patch. 

Link to the current specification. 

We might say we are rolling forward to the latest logical patch, which is the third position for versions earlier than 2.1.100 and the last two digits starting in 2.1.100. 

I think there is a little goofiness in the roll forward of the SDK as we made the transition.

Reply:
Good to know @KathleenDollard. I'll wait to hear from @livarcocc and @dsplaisted on the correct patch. I was assuming it was on the last position, not just the last two digits for newer SDKs.

Reply:
@mairaw See [here](https://github.com/dotnet/core-setup/pull/3711):

> The SDK version rolls up only upon 'minor-patch' versions where the 'minor-patch' version is defined as the last two digits of the 'patch' version.